src/Native/Unix/System.Native/pal_time.c

```diff
@@ -48,75 +48,43 @@ int32_t SystemNative_UTimensat(const char* path, TimeSpec* times)
     return result;
 }
 
-int32_t SystemNative_GetTimestampResolution(uint64_t* resolution)
+uint64_t SystemNative_GetTimestampResolution()
 {
-    assert(resolution);
-
 #if HAVE_MACH_ABSOLUTE_TIME
     mach_timebase_info_data_t mtid;
```

Comment:
Initialize this to zero so that the function returns predictable value in case `mach_timebase_info` fails for some odd reason?

Reply:
Fixed.